LoopKit/Service/Remote/Actions/RemoteAction.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  RemoteAction.swift
+//  LoopKit
+//
+//  Created by Bill Gestrich on 12/25/22.
+//  Copyright © 2022 LoopKit Authors. All rights reserved.
+//
+
```

Comment:
Similar to RemoteCommand. But the associated types are specific to each command. This is in LoopKit as these models will be shared by Caregiver.